templates/highly-efficient-record-processor.cfg.xml.erb

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<highly-efficient-record-processor xmlns="http://docs.openrepose.org/repose/highly-efficient-record-processor/v1.0"
+                                   pre-filter-logger-name="<%= @pre_filter_logger_name %>"
+                                   post-filter-logger-name="<%= @post_filter_logger_name %>"
+                                   service-code="<%= @service_code %>" region="<%= @region %>" data-center="<%= @datacenter %>">
+    <template crush="<%= @template_crush %>">
+<%- if @template.nil? -%>
+    <!-- todo: methodLabel, host.address -->
+    <![CDATA[
+        <?xml version="1.0"?>
+        <event xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
```

Comment:
This is not standard atom entry, btw. We might want to put down a real atom template as a sample.